test/test_excerpt.rb

```diff
@@ -1,22 +1,23 @@
-require 'helper'
+require "helper"
 
 class TestExcerpt < JekyllUnitTest
   def setup_post(file)
-    Document.new(@site.in_source_dir(File.join('_posts', file)), {
-      site:       @site,
-      collection: @site.posts
+    Document.new(@site.in_source_dir(File.join("_posts", file)), {
+      :site => @site,
+      :collection => @site.posts
```

Comment:
Fix committed locally (documenting this so I can keep track of what's done)

There were ~28 of these offenses after rebasing :)